pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -152,9 +205,19 @@ stdenv.mkDerivation rec {
     "-Donnxruntime_USE_FULL_PROTOBUF=OFF"
   ] ++ lib.optionals pythonSupport [
     "-Donnxruntime_ENABLE_PYTHON=ON"
+  ] ++ lib.optionals cudaSupport [
+    "-DFETCHCONTENT_SOURCE_DIR_CUTLASS=${cutlass}"
+    "-Donnxruntime_USE_CUDA=ON"
+    "-Donnxruntime_CUDNN_HOME=${cuda-redist}"
```

Comment:
`cudaPackages.cudnn`

> The original commit sets onnxruntime_CUDNN_HOME to ${cudaPackages.cudnn}, currently it's {cuda-redist}. Either of those should be sufficient, yes?

cudnn and cudatoolkit are separate projects at least at the cmake level; if it should happen upstream build scripts ask for cudnn and cuda to be in the same directory, we must check if there's a way to alter such behaviour

Reply:
`"-Donnxruntime_CUDNN_HOME=${cudaPackages.cudnn}"` also works on my cumpoter

Reply:
I've reverted it back to just `cudaPackages.cudnn`.